.env

```diff
@@ -1,4 +1,4 @@
 PROJECT_ID=artefact-docker-containers
 DOCKER_IMAGE=artefactory-connectors-kit-dev
-DOCKER_TAG=v2.1
+DOCKER_TAG=v2.3
```

Comment:
It's just some minor corrections, you can let the docker_tag to 2.1. Or maybe 2.1.2 if you really want to increase the version ;)

Reply:
Sure! I have made this change now.